test/interfaces/IStakePrizePool.sol

```diff
@@ -0,0 +1,129 @@
+//
+pragma solidity ^0.8.10;
+
+interface IStakePrizePool {
```

Comment:
nit: might be nice to add a comment about how this interface was generated, as done for others.